scripts/share_localhost.py

```diff
@@ -0,0 +1,15 @@
+import qrcode
+import subprocess
+
+# Get the hostname and IP (non loopback) address of the current machine
+cmd = "ifconfig | grep 'inet ' | grep -Fv 127.0.0.1 | awk '{print $2}'"
+output = subprocess.check_output(cmd, shell=True)
+ip_address = output.decode().strip()
+
+# Generate a QR code containing the URL for localhost
+url = f"http://{ip_address}:8000"
```

Comment:
Would be better if it took as argument a name of the bot and did the direct address http://localhost:8000/bots/MyBot 

And no arg is http://localhost:8000/bots/

Reply:
It's not `localhost` since we want to have other devices on the network (not apps on the same device that's running the bot server) connect to the bot by scanning the qr.